bedrock/firefox/firefox_details.py

```diff
@@ -308,36 +309,41 @@ class FirefoxAndroid(_ProductDetails):
         'release': 'builds',
     }
 
+    # OS names defined in bouncer
+    arch_map = {
+        'api-15': 'android',
+        'x86': 'android-x86',
+    }
+
     # Product names defined in bouncer
     product_map = {
         'beta': 'fennec-beta-latest',
         'release': 'fennec-latest',
     }
 
     store_url = settings.GOOGLE_PLAY_FIREFOX_LINK
-    aurora_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
-                       'latest-mozilla-aurora-android')
-    aurora_urls = {
-        'api-9': aurora_url_base + '-api-9/fennec-%s.multi.android-arm.apk',
-        'api-15': aurora_url_base + '-api-15/fennec-%s.multi.android-arm.apk',
-        'x86': aurora_url_base + '-x86/fennec-%s.multi.android-i386.apk',
-    }
-
-    def platforms(self, channel='release'):
-        platforms = self.platform_labels.copy()
-        major_version = int(self.latest_version(channel).split('.', 1)[0])
 
-        # Android Gingerbread (2.3) is no longer supported as of Firefox 48
-        if major_version >= 48:
-            platforms['android'] = _('ARM devices\n(Android 4.0.3+)')
-            platforms['android-x86'] = _('Intel devices\n(Android 4.0.3+ x86 CPU)')
-            del platforms['android-api-9']
+    # Product IDs defined on Google Play
+    store_product_ids = {
+        'nightly': 'org.mozilla.fennec_nightly',  # TBD, Bug 1241114
```

Comment:
Looks like this is handled in `get_download_url` below, and this is hear for future use. Perhaps just a comment here to that affect would suffice?